File: src/main/java/susc/s01/Data/Log/JoinLog/PlayerLog.java
package susc.s01.Data.Log.JoinLog;

import susc.s01.Data.Log.Log;

import java.util.Calendar;
import java.util.UUID;

public record PlayerLog(
        UUID uuid,
        String playerName,
        int joinCount,
        Calendar joinDate,
        Calendar leaveDate
) implements Log {
}

File: src/main/java/susc/s01/Data/Log/JoinLog/PlayerLog.kt
package susc.s01.Data.Log.JoinLog

import susc.s01.Data.Log.Log
import java.util.*

@JvmRecord
data class PlayerLog(
    val uuid: UUID,
    val playerName: String,
    val joinCount: Int,
    val joinDate: Calendar,
    val leaveDate: Calendar
) : Log
